frontend/wasm/src/api.rs

```diff
@@ -552,16 +488,13 @@ impl JsEditorHandle {
 		self.dispatch(message);
 	}
 
-	/// Creates an empty folder at the document root
-	pub fn create_empty_folder(&self) {
-		let message = DocumentMessage::CreateEmptyFolder { container_path: vec![] };
+	// TODO: Replace with initialization system, issue #524
+	pub fn init_app(&self) {
+		// Switch away from, and back to, the Select Tool to make it display the correct hints in the status bar
+		let message = ToolMessage::ActivateTool { tool_type: ToolType::Path };
+		self.dispatch(message);
+		let message = ToolMessage::ActivateTool { tool_type: ToolType::Select };
 		self.dispatch(message);
-	}
-
-	// TODO(mfish33): Replace with initialization system Issue:#524
-	pub fn init_document_bar(&self) {
-		let message = PortfolioMessage::UpdateDocumentBar;
-		self.dispatch(message)
 	}
```

Comment:
@mfish33 this didn't seem to be doing anything so far as I could tell. But perhaps I missed something and caused a regression by deleting it?